testsuite/MDAnalysisTests/analysis/test_wbridge.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import print_function, absolute_import
+from six import StringIO
+
+import MDAnalysis
+from MDAnalysis.analysis.hbonds.wbridge_analysis import WaterBridgeAnalysis
+
+class TestWaterBridgeAnalysis(object):
+    def test_acceptor_water_accepter(self):
```

Comment:
Can you comment what each test case is doing?  It's not easy to read the gro files to figure out which atoms you've shifted in/out

Reply:
@richardjgowers Sorry for the confusion. I have added a short description now.